include/grpc/support/useful.h

```diff
@@ -52,4 +52,13 @@
     b = x;               \
   } while (0)
 
+/** Set the \a n-th bit of \a i */
+#define GPR_BITSET(i, n) (i |= (1u << n))
```

Comment:
can we make it `((*(i)) |= (1u << (n))`
- it makes users pass in a pointer to make it clearer that this mutating
- it gets macro hygiene right to avoid all the things that can happen in text expansion if you don't cover things in parentheses